pysteps/nowcasts/__init__.py

```diff
@@ -1,3 +1,8 @@
 """Implementations of deterministic and ensemble nowcasting methods."""
 
-from pysteps.nowcasts.interface import get_method
+from pysteps.nowcasts.interface import *
+
+try:
+    from dgmr_module_plugin import dgmr
+except ImportError:
+    dgmr = None
```

Comment:
why this block? the dgmr plugin, if installed, will be discovered below in `discover_nowcasts()`

Reply:
oh I see, is it because the function is called `forecast` and it should be added to a module named `dgmr`?  https://github.com/pySTEPS/pysteps-dgmr-nowcasts/blob/c8903bfa689951c555cfb275fb2100e36a683e40/setup.py#L86

Reply:
> oh I see, is it because the function is called `forecast` and it should be added to a module named `dgmr`? https://github.com/pySTEPS/pysteps-dgmr-nowcasts/blob/c8903bfa689951c555cfb275fb2100e36a683e40/setup.py#L86

Also, in the get_method, actually when the plugin is installed, the dgmr is discovered by pysteps and it is possible to 

_dgmr=get_method('dgmr')_.

Though is not quite a nice way, included this block so that when the plugin is installed we could called it directly by doing 

_from pysteps.nowcasts import dgmr_.

Had no idea on how to do that differently and will be very delighted to have some suggestions.

Reply:
I see you removed that part, @Loickemajou , does it mean that you found an alternative solution?

Reply:
> I see you removed that part, @Loickemajou , does it mean that you found an alternative solution?

Hello @dnerini 

I have removed the conflicting dependencies and model weights to ensure that they do not interfere with PySTEPS during execution.

However, I have not yet found an alternative solution and would appreciate any suggestions or insights you might have to help improve the integration process.